cmd/cli/util.go

```diff
@@ -54,19 +55,16 @@ func confirm(stdin io.Reader, stdout io.Writer, s string, tries int) (bool, erro
 	return false, nil
 }
 
-// getPrettyPrintedMeshInfoList a pretty printed list of meshes.
-// If showIndex is true, then a 1-indexed number is prepended before each mesh.
+// getPrettyPrintedMeshInfoList returns a pretty printed list of meshes.
 func getPrettyPrintedMeshInfoList(meshInfoList []meshInfo) string {
-	s := "\nMESH NAME\tMESH NAMESPACE\tCONTROLLER PODS\tVERSION\tSMI SUPPORTED\tADDED NAMESPACES\n"
+	s := "\nMESH NAME\tMESH NAMESPACE\tVERSION\tADDED NAMESPACES\n"
 
 	for _, meshInfo := range meshInfoList {
 		m := fmt.Sprintf(
-			"%s\t%s\t%s\t%s\t%s\t%s\n",
+			"%s\t%s\t%s\t%s\n",
```

Comment:
In the future, we should use a pkg such as github.com/olekukonko/tablewriter for better formatting of tables, so long lines are wrapped around within the column.